README.md

```diff
@@ -47,7 +47,8 @@ repos:
         # "mypy" is the id of a pre-commit hook
         # "types" is the name of your poetry group containing typing dependencies
         # "main" is the automated name associated with the "default" poetry dependencies
-        args: ["--bind", "mypy=types,main"]
+        # `--do-not-add` will update or remove packages, but not add any new one.
+        args: ["--bind", "mypy=types,main", "--do-not-add"]
```

Comment:
Not sure if you want this here. I added it because it seems that this sample config lists all possible option.